src/modules/ToDoList.js

```diff
@@ -0,0 +1,133 @@
+export default class ToDoList {
+  constructor(task, completed, index) {
+    this.task = task;
+    this.completed = completed;
+    this.index = index;
```

Comment:
- [ ] Nice work here, as a requirement you need to set the index property to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). you can set `this.index = index + 1;`